x-pack/plugins/enterprise_search/public/applications/app_search/components/source_engines/source_engines.test.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import '../../../__mocks__/shallow_useeffect.mock';
+
+import { setMockActions, setMockValues } from '../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiCodeBlock } from '@elastic/eui';
+
+import { Loading } from '../../../shared/loading';
+
+import { SourceEngines } from '.';
+
+const MOCK_ACTIONS = {
+  // FlashMessagesLogic
+  dismissToastMessage: jest.fn(),
+  // SourceEnginesLogiuc
```

Comment:
small typo
```suggestion
  // SourceEnginesLogic
```

Reply:
I addressed this in [cc65416](https://github.com/elastic/kibana/pull/96896/commits/cc654166ea55a01918746e7978589df9ca02a644)